packages/edgebridge/README.md

```diff
@@ -0,0 +1,142 @@
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepedgebridge)](https://www.npmjs.com/package/@adobe/react-native-aepedgebridge)
+
+`@adobe/react-native-aepedgebridge` is a wrapper around the iOS and Android Adobe Experience Platform Edge Bridge to allow for integration with React Native applications.
+
+The Adobe Experience Platform Edge Bridge mobile extension enables forwarding of Analytics track events to Experience Platform Edge Network when using the [Experience Platform Mobile SDK](https://developer.adobe.com/client-sdks) and the Edge Network extension. The complete ingestion process occurs in two main steps:
+
+1. Edge Bridge forwards the track event data to Edge Network in a generic data format. 
+2. The generic data is converted into the Experience Data Model (XDM) format, which is a standard experience-driven data schema for Adobe and partner solutions. 
+   * This conversion mapping can be set in the Data Collection datastream associated with the application's mobile property using [Data Prep for Data Collection](https://experienceleague.adobe.com/docs/platform-learn/data-collection/edge-network/data-prep.html).
+  
+   * For more info, refer to [Data Prep for Data Collection Documentation](https://experienceleague.adobe.com/docs/experience-platform/data-prep/home.html).
+
+> **Note**  
+> For new implementations of the Adobe Experience Platform SDK, it's highly recommended to send event data that is already XDM formatted using the [`Edge.sendEvent`](../edge/README.md#api-reference) API instead of converting events from the `MobileCore.trackState` and `MobileCore.trackAction` APIs using Edge Bridge. 
+> 
+> However, in cases where it is not easy to refactor an existing application, the Edge Bridge extension exists as a drop-in solution to send converted `trackState` and `trackAction` events to the Edge Network.
+
+
+## Prerequisites
+
+The Edge Bridge extension has the following peer dependencies, which must be installed prior to installing the Edge Bridge extension:
+- [Core](../core/README.md)
+- [Edge Network](../edge/README.md)
+- [Identity for Edge Network](../edgeidentity/README.md)
+
+## Installation
+
+See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page
+
+Install the `@adobe/react-native-aepedgebridge` package:
+
+```bash
+cd MyReactApp
+npm install @adobe/react-native-aepedgebridge
+```
+## Usage
+
+### Installing and registering the extension with the AEP Mobile Core
+
+Install the `Adobe Experience Platform Edge Network extension` in your mobile property and configure the default Datastream ID by following the steps in the [Edge Network extension documentation](https://developer.adobe.com/client-sdks/documentation/edge-network).
+
+For sending the data to Analytics Report, Add Service `Adobe Analytics` to Datastreams when configurating. 
```

Comment:
I think we can skip this part as the services configuration in datastreams can vary.